origins_shamrock/modules/origins_shamrock_admin/src/Controller/ShamrockDataController.php

```diff
@@ -44,7 +44,17 @@ public static function create(ContainerInterface $container) {
    */
   public function index() {
     $config = $this->config(ShamrockAdminForm::SETTINGS);
+
     $response = new JsonResponse();
+    $response->headers->set('Access-Control-Allow-Headers', [
```

Comment:
This is weird, shouldn't need this as Cors is disabled in the services: https://github.com/dof-dss/nidirect-drupal/blob/development/web/sites/default/default.services.yml#L178